Marlin/Configuration.h

```diff
@@ -1630,6 +1630,24 @@
 
 #define Z_PROBE_LOW_POINT          -2 // Farthest distance below the trigger-point to go before stopping
 
+// For M851 give a range for adjusting the X probe offset
+#ifndef Z_PROBE_OFFSET_XRANGE_MIN
+  #define Z_PROBE_OFFSET_XRANGE_MIN -50
```

Comment:
`ifndef` enables users to augment `Z_PROBE_OFFSET_XRANGE_MIN` without modifying `Configuration.h` (e.g. by adding `-DZ_PROBE_OFFSET_XRANGE_MIN=-20` in their `.ini`

Reply:
Indeed. This could be done for every option. But it's tedious and would explode the size of the configs, so we don't do this. Consider using `config.ini` for overrides instead of adding -D build flags to `*.ini` files going forward.